packages/subiquity_client/test/types_test.dart

```diff
@@ -268,17 +270,21 @@ void main() {
         diskId: '0',
         allowed: [],
         disallowed: [],
+        ptable: null,
       ),
       capability: GuidedCapability.LVM,
       password: '2',
       sizingPolicy: SizingPolicy.ALL,
       resetPartition: false,
+      pin: '0000',
```

Comment:
why are we setting a PIN rather than a passphrase (given that we don't currently support PIN in the UI I mean)?